js/header.js

```diff
@@ -30,3 +35,59 @@ bannerButton.addEventListener('click', function() {
         bannerButton.value = 'Sign up';
     }, 2000);
 });
+
+// generate navbar dropdown
+document.addEventListener('DOMContentLoaded', ()=>{
```

Comment:
You don't need the DOMContentLoaded listener here because this JS file is linked from the end of the document so the body will execute after the content has been loaded